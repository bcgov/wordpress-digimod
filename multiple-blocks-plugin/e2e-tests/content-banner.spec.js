import {
    createNewPost,
    enablePageDialogAccept,
    insertBlock,
    getEditedPostContent
} from '@wordpress/e2e-test-utils';

describe( 'Content banner block', () => {
    jest.setTimeout(60000);

    beforeAll( async () => {
        await enablePageDialogAccept();
    } );
    beforeEach( async () => {
        await createNewPost();
    } );

    it( 'Content banner block should be available', async () => {
        await insertBlock( 'content-banner' );
		// Check if block was inserted
        
		expect( await page.$( '[data-type="multiple-blocks-plugin/content-banner"]' ) ).not.toBeNull();

		expect( await getEditedPostContent() ).toMatchSnapshot();
	} );
} );