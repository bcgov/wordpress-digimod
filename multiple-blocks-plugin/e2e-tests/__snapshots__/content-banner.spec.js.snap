// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content banner block Content banner block should be available 1`] = `
"<!-- wp:multiple-blocks-plugin/content-banner -->
<div class=\\"wp-block-group alignwide\\"><div id=\\"main-content-anchor\\" class=\\"content-banner-wrapper container\\"><div class=\\"row middle-xs\\"><!-- wp:multiple-blocks-plugin/dm-content-banner-content -->
<div class=\\"col-sm-12 col-md-6\\"><div><!-- wp:post-title {\\"level\\":1} /--><div class=\\"content-banner-short-text\\"></div></div></div>
<!-- /wp:multiple-blocks-plugin/dm-content-banner-content -->

<!-- wp:multiple-blocks-plugin/dm-content-banner-image -->
<div class=\\"col-sm-12 col-md-6\\"><!-- wp:image -->
<figure class=\\"wp-block-image\\"><img alt=\\"\\"/></figure>
<!-- /wp:image --></div>
<!-- /wp:multiple-blocks-plugin/dm-content-banner-image --></div></div></div>
<!-- /wp:multiple-blocks-plugin/content-banner -->"
`;
